src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started its work.");

```

Comment:
The message 'Excavator started its work.' could be more informative. According to the checklist, it should indicate what type of `Machine` is working right now. Consider specifying that it is an `Excavator` machine.